command/operator_debug.go

```diff
@@ -961,7 +961,7 @@ func (c *OperatorDebugCommand) collectPprof(path, id string, client *api.Client,
 			return // only exit on 403
 		}
 	} else {
-		filename := fmt.Sprintf("profile_%d.prof", interval)
+		filename := fmt.Sprintf("profile_%04d.prof", interval)
```

Comment:
Read the commit message but which console output are we trying to match here? A little out of the loop.

Reply:
These lines: https://github.com/hashicorp/nomad/pull/11938/files#diff-910fd48022b1afbffeb20512a6a5a28757a04addcc70cedccee8294582455135R917-R918

```go
name = fmt.Sprintf("%04d", pprofIntervalCount)
c.Ui.Output(fmt.Sprintf("    Capture pprofInterval %s", name))
```